llvm/test/CodeGen/AArch64/exception-handling-windows-elf.ll

```diff
@@ -0,0 +1,42 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc -mtriple=aarch64-pc-windows-elf < %s | FileCheck %s
+; Make sure windows elf does not assert with exceptions
```

Comment:
should state that this is a MCJIT workaround (which means it can retire when MCJIT is removed)